packages/deployer/hardhat.config.ts

```diff
@@ -71,6 +71,7 @@ const config: HardhatUserConfig = {
 
     dev: getNetwork('http://localhost:8545'),
     rarb: getNetwork('https://rinkeby.arbitrum.io/rpc'),
+    garbn: getNetwork('https://goerli-rollup.arbitrum.io/rpc'),
```

Comment:
why "n" ? nitro is now an integral feature of arbitrum, not anything special (they don't call the network "nitro" either_